### PR TITLE
New package: dremin.RetroBar version 1.15

### PR DESCRIPTION
**Pull request by Chartix**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/152029)

### CODE REVIEW

**Comment by Chartix** [comment]
@microsoft-github-policy-service agree

File: manifests/d/dremin/RetroBar/1.15/dremin.RetroBar.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: dremin.RetroBar
+PackageVersion: "1.15"
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: RetroBar.exe
+  PortableCommandAlias: retrobar
+Installers:
```
Comment (stephengillie) [inline manifests/d/dremin/RetroBar/1.15/dremin.RetroBar.installer.yaml:11]:
```suggestion
Dependencies:
  PackageDependencies:
     - PackageIdentifier: Microsoft.DotNet.DesktopRuntime.6
Installers:
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
Manual Validation ended with: 
image

File: manifests/d/dremin/RetroBar/1.15/dremin.RetroBar.installer.yaml (hunk 1 above)
Comment (stephengillie) [reply manifests/d/dremin/RetroBar/1.15/dremin.RetroBar.installer.yaml:11]: With the dependency, this package appears to install and run normally: 
image

**Review by Chartix — COMMENTED** [review]
Adds necessary .NET Dependency

**Review by Chartix — COMMENTED** [review]
Adds .NET 6 Dependency

**Comment by stephengillie** [comment]
Verify: 
image


Footnote: Installing this application might lose the current taskbar icons. Before: 
image

After: 
image

Links to Windows Store, Settings, and Defender aren't in the Quick Launch bar after install.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**